src/main/java/com/hugudungs/hugupjigup/data/entity/board/FreeText.java

```diff
@@ -0,0 +1,33 @@
+package com.hugudungs.hugupjigup.data.entity.board;
+
+import com.hugudungs.hugupjigup.data.entity.user.User;
+import jakarta.persistence.*;
+import lombok.*;
+import java.util.Date;
+
+@Entity
+@Table(name = "free_text")
```

Comment:
```suggestion
@Table(name = "free")
```
free_text 보다는 free로 이름 바꾸는게 더 좋아보여
파일명도 Free로 같이 리팩토링해줘